file_share_through_db.py

```diff
@@ -0,0 +1,55 @@
+"""
+script that saves xdmod's configuration to a db in order to share it
+with cron jobs that are being run in separate pods (the shredder and
+the ingestor kubernetes cron jobs)
+"""
+import os
+import json
+import moc_db_helper_functions as moc_db
+
+
+def write_file_to_db(cursor, filename, script):
```

Comment:
Should this also live in `moc_db_helper_functions.py`? Or is it never used anywhere else?

Reply:
It is only used here.  When I rework xdmod-init i'll add it then.